_data/members.yaml

```diff
@@ -18,3 +20,12 @@
     twitter: rgausnet
     website: http://rgaus.net
     email: rsg1egoman@gmail.com
+
+- name: Peter B Smith
+  img: http://peterbsmith.net/img/profile.png
```

Comment:
Use Gravatar plz! 

Reasoning: I don't want the members page to break because you change your personal website structure somewhere down the road